bedrock/base/templates/base-new.html

```diff
@@ -0,0 +1,149 @@
+{% from "macros.html" import share_cta with context %}
```

Comment:
Yeah, I'm not adverse to a new base template at all - I was more thinking of using shared includes for the bits that don't really change, it avoid having to update everything 3x.